gateway/ln-gateway/tests/integration_tests.rs

```diff
@@ -290,14 +291,19 @@ async fn test_can_change_routing_fees() -> anyhow::Result<()> {
             })
             .await;
 
+            user_client
+                .get_first_module::<LightningClientModule>()
+                .set_active_gateway(&gateway.gateway.gateway_id)
+                .await?;
+
             // Create test invoice
             let invoice_amount = sats(250);
             let invoice = other_lightning_client.invoice(invoice_amount, None).await?;
 
             let gateway = gateway.remove_client_hack(&fed).await;
             pay_valid_invoice(invoice, &user_client, &gateway).await?;
```

Comment:
Could we, in preparation for deprecating the active gateway concept, add a gateway id argument to that function here? I think internally it already calls a function that accepts a gateway as argument.

Reply:
Good call. Done